packages/app-store/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -119,37 +148,59 @@ export default function RoutingForms({
         <FormActionsProvider appUrl={appUrl}>
           <div className="-mx-4 md:-mx-8">
             <div className="mb-10 w-full px-4 pb-2 sm:px-6 md:px-8">
-              {!forms?.length ? (
-                <EmptyScreen
-                  Icon={GitMerge}
-                  headline={t("create_your_first_form")}
-                  description={t("create_your_first_form_description")}
-                  buttonRaw={<NewFormButton />}
-                />
-              ) : null}
-              {forms?.length ? (
+              <div className="flex">
+                <TeamsFilter />
+              </div>
+              <FilterResults
```

Comment:
Abstracts EmptyState, No Filter results state and loading state into FilterResults.
FilterResults also enforces that the result from queryRes should have `totalCount` and `filtered` props